aws-android-sdk-mobile-client/src/main/java/com/amazonaws/mobile/client/AWSMobileClient.java

```diff
@@ -1325,6 +1325,14 @@ public void run() {
                         detectedContinuation = signInChallengeContinuation;
                         signInCallback = new InternalCallback<SignInResult>(callback);
                         break;
+                    case CUSTOM_CHALLENGE:
+                        signInChallengeContinuation.setChallengeResponse("ANSWER", signInChallengeResponse);
```

Comment:
I found this documentation about implementing [`CUSTOM_CHALLENGE`](https://docs.amazonaws.cn/en_us/cognito/latest/developerguide/amazon-cognito-user-pools-authentication-flow.html#Custom-authentication-flow-and-challenges). But, it doesn't mention any `"ANSWER"` response. How did you arrive at this key?

Reply:
@jamesonwilliams  I found this in the the[ aws-sdk-ios](https://github.com/aws-amplify/aws-sdk-ios/blob/2508bf8aecc3cef4e47ebba2873cb0822c2caa33/AWSAuthSDK/Sources/AWSMobileClient/AWSMobileClientExtensions.swift#L702).

Reply:
Aha! 🚢